File: Homeworks/C#1/05.ConditionalStatementsHomework/08.DigitAsWord/DigitAsWord.cs
using System;

class DigitAsWord
{
    static void Main()
    {
        Console.Write("Please, enter digit from 0 to 9: ");
        int digit = int.Parse(Console.ReadLine());

        switch (digit)
        {
            case 0: Console.WriteLine("zero"); break;
            case 1: Console.WriteLine("one"); break;
            case 2: Console.WriteLine("two"); break;
            case 3: Console.WriteLine("three"); break;
            case 4: Console.WriteLine("four"); break;
            case 5: Console.WriteLine("five"); break;
            case 6: Console.WriteLine("six"); break;
            case 7: Console.WriteLine("seven"); break;
            case 8: Console.WriteLine("eight"); break;
            case 9: Console.WriteLine("nine"); break;
            default: Console.WriteLine("not a digit"); break;
        }
    }
}

File: Homeworks/JavaScriptFundamentals/03.ConditionalStatements/05.DigitAsWord/DigitAsWord.js
/*
 Write script that asks for a digit and depending on
 the input shows the name of that digit (in English)
 using a switch statement.
 */

var jsConsole;
var button = document.getElementById('translate');

button.onclick = translateDigitAsWord;

function translateDigitAsWord() {
    var input = jsConsole.readInteger('#input-number');

    switch (input) {
        case 0:
            jsConsole.writeLine('The input digit is Zero');
            break;
        case 1:
            jsConsole.writeLine('The input digit is One');
            break;
        case 2:
            jsConsole.writeLine('The input digit is Two');
            break;
        case 3:
            jsConsole.writeLine('The input digit is Three');
            break;
        case 4:
            jsConsole.writeLine('The input digit is Four');
            break;
        case 5:
            jsConsole.writeLine('The input digit is Five');
            break;
        case 6:
            jsConsole.writeLine('The input digit is Six');
            break;
        case 7:
            jsConsole.writeLine('The input digit is Seven');
            break;
        case 8:
            jsConsole.writeLine('The input digit is Eight');
            break;
        case 9:
            jsConsole.writeLine('The input digit is Nine');
            break;
    }
}